modules/test/playwright/pages/style-book-web/StyleBooksPage.ts

```diff
@@ -58,4 +60,42 @@ export class StyleBooksPage {
 
 		await this.page.getByRole('button', {name: 'Delete'}).click();
 	}
+
+	async editStyleBook(styleBookName: string) {
+		await this.page
+			.locator(
+				'input[name=_com_liferay_style_book_web_internal_portlet_StyleBookPortlet_keywords][type=search]'
```

Comment:
This button is also in the deleteStyleBook method. Let's create a private field for that.